utils/_context/_scenarios/__init__.py

```diff
@@ -756,6 +756,12 @@ class _Scenarios:
         scenario_groups=[ScenarioGroup.APPSEC, ScenarioGroup.APPSEC_RASP],
     )
 
+    graphql = EndToEndScenario(
```

Comment:
Can you use the `GRAPHQL_APPSEC` scenario? (yes, there is appsec in its name, but that was an error, it can test all graphQL related features).